mkdocs_static_i18n/plugin.py

```diff
@@ -302,6 +302,7 @@ def on_config(self, config, **kwargs):
         """
         Enrich configuration with language specific knowledge.
         """
+        if 'skip_i18n' in config: return
```

Comment:
what is this `skip_i18n` option you're referring to?

Reply:
The deepcopy of the config object for the languages also includes the i18n config itself. When running events on the language config it also triggers the event handlers of the i18n config again getting in an endless loop. To avaoid that, I added this flag to the copied config:
https://github.com/Maddin-619/mkdocs-static-i18n/blob/f134d66f3fa43dab16fcbabb2d7c4e84adb9751b/mkdocs_static_i18n/plugin.py#L382

Reply:
More than the deepcopy itself it seems like it's the fact that `with-pdf` does call other plugin's `post_build` just like `static-i18n` does with other events.

So the proper way to handle that would be to implement something more generic to make sure we never run more than once. I don't like these kind of flags tbh.

Reply:
I don't like it either. This was a very quick solution to get it running and make a first proposal.
I can imagine that extending the plugin collection class with a event run methode witch accepts a list of plugins the event should trigger would be a cleaner solution.